decred/decred/dcr/rpc.py

```diff
@@ -53,6 +53,124 @@ def call(self, method, *params):
             raise Exception("%s error: %r" % (method, res["error"]))
         return res["result"]
 
+    def existsAddress(self, address):
+        """
+        Test for the existence of the provided address.
+
+        Args:
+            address (string): The address to check.
+
+        Returns:
+            bool: True if address exists.
+        """
+        return self.call("existsaddress", address)
+
+    def existsAddresses(self, addresses):
+        """
+        Test for the existence of the provided addresses in the blockchain or
+            memory pool.
+
+        Args:
+            addresses (list(str)) The addresses to check.
+
+        Returns:
+            string: Bitset of bools showing if addresses exist or not.
```

Comment:
Let's do `list(bool)`. Should be something like

```
mask = int(self.call("existsaddresses", addresses))
[bool(mask & 1 << n) for n in range(len(addresses))]
```

but I haven't tested.

Same for `existsLiveTickets`, `existsMempoolTxs`, and `existsMissedTickets`.

Reply:
It's interesting that none of these can be called with lists above 8 hashes.